plugins/removeComments.js

```diff
@@ -5,6 +5,17 @@ const { detachNodeFromParent } = require('../lib/xast.js');
 exports.name = 'removeComments';
 exports.description = 'removes comments';
 
+/**
+ * If a comment matches one of the following patterns, it will be
+ * preserved by default. Particularly for copyright/license information.
+ */
+const DEFAULT_PRESERVE_PATTERNS = [
+  /^!/,
```

Comment:
Does anyone here know the motivation for the `/^!/` check?
With the new preserve patterns param, perhaps that one can be removed in v4?

Reply:
`!` is a convention for preservable comments like license. All minifiers rely on it.
No need to check for keywords with it.

Reply:
Interesting, is this convention defined anywhere, even if only informally? I'd just like to make a note and reference it in the documentation and JSDocs? I've tried searching for this, but don't see a resource that makes note of this. 🤔

In my opinion, it makes sense to retain the keywords as well if this isn't a defined standard. In the worst-case scenario, it just adds a bit of redundancy while processing, what do you think?